projects/image-rs/Dockerfile

```diff
@@ -48,3 +48,19 @@ RUN ../build.sh
 RUN  scrpit=$(find /root/.cargo/registry/src/ -name update-libfuzzer.sh) && $scrpit 3c7960cba19ed926b8e86b4c619e81c0f7da4d15
 
 RUN ../build.sh
+
+# Default corpus.
+RUN mkdir /corpus
```

Comment:
We do not need empty corpus -- no input corpus is allowed for libFuzzer

Reply:
done